gen/armor.py

```diff
@@ -147,6 +150,7 @@ def generate(self) -> None:
         if self.game.blue.faction.has_jtac:
             n = "JTAC" + str(self.conflict.blue_cp.id) + str(self.conflict.red_cp.id)
             code = 1688 - len(self.jtacs)
+            freq = MHz(30 + len(self.jtacs) * 10)
```

Comment:
This needs to be allocated using the `RadioRegistry` so it doesn't end up reused by other aircraft. It will also prevent overlap with map beacons like ATCs and nav beacons.

I'm guessing that the generic `alloc_uhf` is correct here. As-is this would only be usable by a small number of aircraft. Neither the hornet, viper, nor tomcat could talk to this JTAC. I think the only aircraft capable of 30MHz are the A-10, Ka-50, and UH-1?

Reply:
I have updated the code to use `RadioRegistry` for JTAC radio frequency!

Reply:
Thanks! I set the checks to rerun (it needs a project member approval for anyone that hasn't merged a PR yet). I'll re-review and hopefully merge tomorrow (I'm in the US so it's late here).